src/test/java/jewellery/inventory/integration/AuthenticatedIntegrationTestBase.java

```diff
@@ -66,9 +66,7 @@ void setup() {
     User adminUser = createTestUserWithId();
     setupMockSecurityContext(adminUser);
     setupTestRestTemplateWithAuthHeaders();
-    User adminDifferent = createUserInDatabase(UserTestHelper.createDifferentTestUserRequest());
-    setupMockSecurityContext(adminDifferent);
-    setupTestRestTemplateWithAuthHeaders();
+    adminUser.setId(createUserInDatabase(UserTestHelper.getTestUserRequest(adminUser)).getId());
```

Comment:
I guess that's why... I think we should consider putting the authenticated user as a different (adminUser) than the testUser we use to test other user related tests.